rest-api/offline.yaml

```diff
@@ -28,7 +28,6 @@ handlers:
   login: admin
 - url: /offline/.*
   script: offline.main.app
-  login: admin
```

Comment:
Just had to think about this for a bit -- as long as we keep `admin` as a requirement for the `_ah/...` routes, I think we're okay.